src/main/java/com/mwt/tests/BlackBox.java

```diff
@@ -1,545 +0,0 @@
-package com.mwt.tests;
```

Comment:
The unit tests below look great, thanks for adding them. However, we should keep the BlackBox tests and have it run in the build server. The point of these tests is to make sure any language-specific interface produces same output as other versions of the exploration libraries. For example, our [C++](https://github.com/multiworldtesting/explore-cpp) and [C#](https://github.com/multiworldtesting/explore-csharp) code are already doing this in appveyor to make sure their exploration is correct. I previously wanted to set this up for Java but haven't got around to it. Basically, the steps needed in the build server are:
1. Clone the [BlackBox test project](https://github.com/multiworldtesting/explore-test) and either the C++ or C# version and build it into an .exe. For example, the appveyor script for C# is [here](https://github.com/multiworldtesting/explore-csharp/blob/master/appveyor.yml).
2. Trigger the BlackBox tests and compare outputs. This would require that I modify the test project to include the Java BB interface. I'll try to do this but might be a bit later this week as we're in crunch time right now for MWT system. I don't think this should block you from releasing to Maven though because I've manually verified that the results match previously and the core code is unchanged.

One more thing, now that we have Travis set up, we should probably keep BB tests running in both servers. A downside (which I just realized) is that the BB test project is written in C# so we may not be able to run it in Travis. Do you have any suggestions? I'm ok with just running tests in Appveyor for now as well.